android/build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.0'
+        classpath 'com.android.tools.build:gradle:3.3.2'
```

Comment:
To fix a build issue (see https://github.com/wix/react-native-navigation/issues/4757#issuecomment-468133753)